lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -211,14 +221,12 @@ class DecomposeAtenTanhBackwardOp
     Value gradMulTanhSquare = rewriter.create<AtenMulTensorOp>(
         loc, tensorType, tanhSquare, gradOutput);
 
-    Value alpha =
-        rewriter.create<ConstantFloatOp>(loc, rewriter.getF64FloatAttr(1));
-    Value newGrad = rewriter.create<AtenSubTensorOp>(
-        loc, tensorType, gradOutput, gradMulTanhSquare, alpha);
+    Value newGrad = createAtenSubTensorOp(rewriter, loc, tensorType, gradOutput,
```

Comment:
nit: `createAtenSubTensorOp` seems to be too specific but strictly speak we also created ops other than just `atenSubTensorOp`. Maybe just `createTensorSub` or `createTensorSubPattern` to indicate the generated ops are for tensors subtraction? And speaking of naming, `createAtenSum` seems to be better called `createSumAlongDimension` or `createSumAlongDimensionPattern`. What do you think?

Reply:
Yes, the naming conventions can be improved. I can change the names as suggested in this PR.